packages/edit-post/src/components/header/post-publish-button-or-toggle.js

```diff
@@ -61,9 +61,9 @@ export function PostPublishButtonOrToggle( {
 		( isPending && ! hasPublishAction && ! isSmallerThanMediumViewport )
 	) {
 		component = IS_BUTTON;
-	} else if ( isSmallerThanMediumViewport ) {
+	} else if ( isSmallerThanMediumViewport && ! isPublishSidebarOpened ) {
```

Comment:
Here we make sure the button only acts as a toggle if the sidebar _feature_ is enabled _and_ the actual sidebar is not yet _opened_.

Reply:
Should a button be concerned with all that? Sounds like we might want to extract the layout management logic into another, higher-level component.